source/exploring_catalog_and_datasets/02_looking_up_a_dataset/visualizing_data.rst

```diff
@@ -0,0 +1,198 @@
+* |icon-fullscreen| to view your map in fullscreen mode
+* zoom in |icon-zoom-in| and zoom out |icon-zoom-out| buttons
+* localize |icon-localize| button to focus the map on your current location
+
+.. admonition:: Caution
+   :class: caution
+
+   Maps displayed in the Map tab cannot be customized, except from choosing another basemap when possible. To be able to customize a map, or cross data from different datasets, as well as to save a created map, use the :doc:`Map Builder <../../creating_maps_charts/01_creating_advanced_maps/overview-of-map-builder>`.
+
+Visualizing data through a chart
+--------------------------------
+
+The chart visualization is accessible from the Analyze tab.
+
+.. image:: images/analyze_tab.png
+   :alt: Chart visualization
+
+It displays a default chart, set by the portal administrators. It is however possible, and encouraged, to configure other charts to make other analysis than the default one.
+
+Above the chart are displayed a variety of buttons and options to configure the chart:
+
+.. image:: images/chart1.png
+   :alt: First line of chart configuration
+
+- The first line is to configure the base of the chart.
+
+  - X axis: to choose which field to use as X axis (horizontal line)
+  - Max. number of points: to choose how many points should be represented on the chart
+  - Default ordering: for points to be displayed in a decreasing order in the chart
+
+.. image:: images/chart2.png
+   :alt: Second line of chart configuration
+
+- The second line is to define how the different series are represented.
+
+  - Break down series: to separate the series into several groups
+  - Stack: for series to be stacked one on top of the other
+  - Add a serie: to display other series on the chart
+
+.. image:: images/chart3.png
+   :alt: Third line of chart configuration
+
+- The third and last line is to configure series separately.
+
+  - a scroll-down menu: to choose a chart type for the series
+  - a color or color palette: to choose the color of the series
+  - Y axis: to choose which field to use as Y axis (vertical line)
+  - Use for ordering: for points to be displayed in a decreasing order in the chart, based on the chosen series
+
+.. admonition:: Caution
+   :class: caution
+
+   Charts displayed in the Analyze tab can be modified. However, these modifications cannot be saved. To be able to save a created chart, as well as to cross data from different datasets, use the :doc:`Chart Builder <../../creating_maps_charts/02_creating_advanced_charts/analyze>`.
+
+Visualizing data through images
+-------------------------------
+
+The images visualization is accessible from the Images tab.
+
+.. image:: images/images_tab.png
+   :alt: Images visualization
+
+It displays the thumbnails of the images of the dataset. When clicking on a thumbnail, the metadata of the image are displayed. Below the metadata is also displayed a Download image button, to download the selected image.
```

Comment:
> Below the metadata is also displayed a Download image button, to download the selected image.

Below the metadata **there is** a Download image button, to download the selected image.